cypress/support/step_definitions/annual-review/scheduler/the_reminder_email_for_string_weeks_is_sent_to_eligiable_providers.js

```diff
@@ -31,7 +34,41 @@ Then("the reminder email for {string} weeks is sent to eligible providers", asyn
         },
       },
     }).then((result) => {
+      cy.wrap(result.length).as("emailsSent");
       cy.expect(result.length).to.be.gt(0);
-    })
-  })
+    });
+  });
+});
+
+Then("the weeklyReminder emails were not re-sent", async () => {
+  cy.task("db", {
+    operation: "list.findFirst",
+    variables: {
+      where: {
+        reference: "SMOKE",
+      },
+    },
+  }).then((list) => {
+    cy.task("db", {
+      operation: "event.findMany",
+      variables: {
+        where: {
+          type: "REMINDER",
+          annualReviewEmailType: "weeklyUnpublish",
+          AND: [
+            {
+              jsonData: {
+                path: ["reference"],
+                equals: list.jsonData.currentAnnualReview.reference,
+              },
+            },
+          ],
+        },
+      },
+    }).then((result) => {
+      cy.get("@emailsSent").then(($emailsSent) => {
+        cy.expect($emailsSent).to.be.equal(result.length);
+      });
```

Comment:
Nice use of a Cypress alias. Why `$emailsSent` instead of `emailsSent`?

Reply:
Mixing up my Cypress practices. when you .get an element in cypress, they want you to use something like $button. Couldn't tell you why. jQuery is gone but never forgotten. :)